src/Sulu/Bundle/AdminBundle/Admin/View/ResourceTabViewBuilderInterface.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Sulu\Bundle\AdminBundle\Admin\View;
 
-interface ResourceTabViewBuilderInterface extends ViewBuilderInterface
+interface ResourceTabViewBuilderInterface extends TabViewBuilderInterface
```

Comment:
This part was already a bit messy, but there is a reason we didn't extend interface in other places (e.g. `PreviewFormViewBuilderInterface` does not extend the `FormViewBuilderInterface`). That is like that because  [return type covariance](https://www.php.net/manual/de/migration74.new-features.php#migration74.new-features.core.type-variance) has not been supported before PHP 7.4, and we are still supporting older versions.

This is also the only reason we are using traits here, they would not be necessary if we could already add the correct return type in the first place.